### ISSUE TITLE
<nav> in menu should have a aria-label="main menu navigation"

### ISSUE THREAD

**Issue by bryanjonker-illinois** [web-illinois/toolkit#48]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute
